axelrod/strategies/titfortat.py

```diff
@@ -642,10 +642,10 @@ def strategy(self, opponent: Player) -> Action:
 class Alexei(Player):
     """
     Plays similar to Tit-for-Tat, but always defect on last turn.
-    
+
     Names:
 
-    - Alexei's Strategy: [LessWrong2011]_    
+    - Alexei's Strategy: [LessWrong2011]_
```

Comment:
Is this correct? There's already a strategy called Alexei's strategy, and the name of this one is EugineNier.